pkg/server/apiextensions.go

```diff
@@ -103,9 +102,6 @@ func (c *apiBindingAwareCRDLister) List(ctx context.Context, selector labels.Sel
 	}
 	for _, obj := range objs {
 		apiBinding := obj.(*apisv1alpha1.APIBinding)
-		if !conditions.IsTrue(apiBinding, apisv1alpha1.InitialBindingCompleted) {
```

Comment:
`apiBindingAwareCRDLister.List()` shows the CRDs that are either present from literal CRD objects, projected from `kcp`, or present from `APIBindings` - but if the binding has not been bound yet, it is not providing any APIs in the workspace, right? So what is the meaning from removing this block?

Reply:
referring to @sttts as this is resulting from pair programming.

Reply:
> but if the binding has not been bound yet, it is not providing any APIs in the workspace, right?

The condition is not the source of thruth, the bound resource slice is. The binding can be partial at any point in time (imagine a resource is added to an export, but a CRD is in place leading to a conflict). Making this logic dependent on a condition it not helpful, i.e. we should always just consider bound resources.